syncstorage/tests/functional/test_storage.py

```diff
@@ -780,15 +780,20 @@ def test_get_collection_ttl(self):
         self.assertEquals(len(res.json), 0)
 
     def test_batch(self):
-        # This can't be run against a live server
```

Comment:
It now *can* be run against a live server, if it returns the proper config info \o/